src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -99,6 +99,20 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, string meth
                         //TODO figure out how to stich out more frames and, in particular what happens when real wasm is on the stack
                         var top_func = args?["callFrames"]?[0]?["functionName"]?.Value<string>();
 
+                        if (top_func == "mono_wasm_add_lazy_load_files")
+                        {
+                            var loaded_assemblies = await SendMonoCommand(sessionId, MonoCommands.GetLazyLoadedFiles(), token);
```

Comment:
Ensure that `loaded_assemblies.IsOk == true`.
nit: Maybe even rename that to res? and `res_val` to `loaded_assemblies`?